### ISSUE TITLE
Error in defalias checker

### ISSUE THREAD

**Issue by alphapapa** [purcell/package-lint#129]
In my new package https://github.com/alphapapa/ts.el, `package-lint` raises a spurious error due to parsing a `defalias` form inside a macro:

```
Debugger entered--Lisp error: (wrong-type-argument symbolp ,alias-name)
  symbol-name(,alias-name)
  package-lint--check-defalias((defalias ',alias-name ',accessor-name))
  package-lint--check-objects-by-regexp("(\\(?:def\\(?:\\(?:var\\)?alias\\)\\)\\s-" package-lint--check-defalias)
  package-lint--check-all()
  package-lint-buffer()
  package-lint-current-buffer()
  funcall-interactively(package-lint-current-buffer)
  call-interactively(package-lint-current-buffer record nil)
  command-execute(package-lint-current-buffer record)
  helm-M-x(nil #("package-lint-current-buffer" 0 27 (match-part "package-lint-current-buffer")))
  funcall-interactively(helm-M-x nil #("package-lint-current-buffer" 0 27 (match-part "package-lint-current-buffer")))
  call-interactively(helm-M-x nil nil)
  command-execute(helm-M-x)
```

**Comment by Fanael** [comment]
Thanks for spotting this, should be fixed now.

**Comment by alphapapa** [comment]
Thanks.
